apps/aehttp/test/aehttp_contracts_SUITE.erl

```diff
@@ -603,133 +868,135 @@ environment_contract(Config) ->
+                                    <<"nested_value">>, <<"(42)">>),
     ct:pal("Nested value ~p\n", [NestedValue]),
 
     %% Gas price.
     ct:pal("Calling call_gas_price\n"),
-    {GPValue,_} = call_func(NodeName, BPubkey, BPrivkey,
-			    EncodedContractPubkey,
-			    HexCode, <<"call_gas_price">>, <<"()">>),
+    {GPValue,_} = call_compute_func(NodeName, BPubkey, BPrivkey,
+                                    EncodedContractPubkey,
+                                    <<"call_gas_price">>, <<"()">>),
     #{<<"value">> := GasPrice} = decode_data(<<"int">>, GPValue),
     ct:pal("Gas price ~p\n", [GasPrice]),
 
     %% Account balances.
     ct:pal("Calling get_balance twice\n"),
-    {BBalValue,_} = call_func(NodeName, BPubkey, BPrivkey,
-			      EncodedContractPubkey,
-			      HexCode, <<"get_balance">>,
-			      args_to_binary([BPubkey])),
+    {BBalValue,_} = call_compute_func(NodeName, BPubkey, BPrivkey,
+                                      EncodedContractPubkey,
+                                      <<"get_balance">>,
+                                      args_to_binary([BPubkey])),
     #{<<"value">> := BBalance} = decode_data(<<"int">>, BBalValue),
     ct:pal("Balance B ~p\n", [BBalance]),
 
-    {DBalValue,_} = call_func(NodeName, BPubkey, BPrivkey,
-			      EncodedContractPubkey,
-			      HexCode, <<"get_balance">>,
-			      args_to_binary([DPubkey])),
+    {DBalValue,_} = call_compute_func(NodeName, BPubkey, BPrivkey,
+                                      EncodedContractPubkey,
+                                      <<"get_balance">>,
+                                      args_to_binary([DPubkey])),
     #{<<"value">> := DBalance} = decode_data(<<"int">>, DBalValue),
     ct:pal("Balance D ~p\n", [DBalance]),
 
     %% Block hash.
     ct:pal("Calling block_hash\n"),
-    {BHValue,_} = call_func(NodeName, CPubkey, CPrivkey,
-			    EncodedContractPubkey,
-			    HexCode, <<"block_hash">>, <<"(21)">>),
+    {BHValue,_} = call_compute_func(NodeName, CPubkey, CPrivkey,
+                                    EncodedContractPubkey,
+                                    <<"block_hash">>, <<"(21)">>),
     #{<<"value">> := BlockHash} = decode_data(<<"int">>, BHValue),
     ct:pal("Block hash ~p\n", [BlockHash]),
 
     %% Coinbase.
     %% ct:pal("Calling coinbase\n"),
-    %% call_func(NodeName, CPubkey, CPrivkey, EncodedContractPubkey, HexCode,
-    %% 	      <<"coinbase">>, <<"()">>),
+    %% call_compute_func(NodeName, CPubkey, CPrivkey, EncodedContractPubkey,
+    %%                <<"coinbase">>, <<"()">>),
 
     %% Block timestamp.
     %% ct:pal("Calling timestamp\n"),
-    %% call_func(NodeName, CPubkey, CPrivkey, EncodedContractPubkey, HexCode,
-    %% 	      <<"timestamp">>, <<"()">>),
+    %% {TimeStampValue,_} = call_compute_func(NodeName, CPubkey, CPrivkey,
+    %%                                     EncodedContractPubkey,
+    %%                                     <<"timestamp">>, <<"()">>),
```

Comment:
It looks like there is [a test for timestamp](https://github.com/aeternity/epoch/blob/e9d82c02826228ebf1e47246252d93ab8c0a7717/apps/aevm/test/aevm_eeevm_tests.erl#L714). Maybe this line shall be enabled?

Reply:
Timestamp doesn't seem to work but coinbase does so that has been included.